### DISCUSSION TITLE
主页的 [交流] 页面上的大部分链接都噶了

### DISCUSSION THREAD

**Discussion by luochen1990** [howmanybots/onebot#258] [问答] [answered]
这个页面 https://onebot.dev/chat.html  上面的大部分链接都噶了。。

<img width="466" alt="image" src="https://github.com/user-attachments/assets/60a82d7b-7584-457a-85d9-455a24e2f83d">

**Comment by stdrc** [comment, marked as answer]
噢，1bot.dev 域名到期没再续费了，忘记改这些链接了😇😇稍后修复
